### PR TITLE
Update mitigation for foxnews

### PR DESCRIPTION
**Pull request by GuiltyDolphin**
https://app.asana.com/0/1200223097357040/1202121182005411/f

### CODE REVIEW

File: features/tracker-allowlist.json [hunk 1]
```diff
@@ -395,14 +395,42 @@
                     }
                 ]
             },
-            "segment.io": {
+            "segment.com": {
                 "rules": [
+                    {
+                        "rule": "cdn.segment.com/analytics-next/bundles/130.bundle.487014ebc6f0bb448b6e.js",
```
Comment (kdzwinel) [inline features/tracker-allowlist.json:401]: this seems fragile - `/130.bundle.487014ebc6f0bb448b6e` - it suggests that they are regularly updating names of those libraries . Making rule a bit broader e.g. `cdn.segment.com/analytics-next/bundles/*.js` would reduce number of rules and be less fragile IMO.

Comment (GuiltyDolphin) [reply features/tracker-allowlist.json:401]: @kdzwinel there are various requests that could be made that we don't want to match here - e.g., `segment.com/analytics-next/bundles/ajs-destination.bundle.a6950cf6bd0c8b0b0e97.js`. This should be fine as a temporary mitigation - we can review more general rules in investigations.

File: features/tracker-allowlist.json [hunk 2]
```diff
@@ -395,14 +395,42 @@
...
                     {
                         "rule": "cdn.segment.com/analytics.js/v1/4I2fAtE5IxTlczNT3F91OckHtI5wfR7g/analytics.min.js",
                         "domains": [
                             "foxnews.com"
                         ],
-                        "reason": "user cannot access create account page from sign up page"
+                        "reason": "When on login page, clicking 'create account' has no effect."
+                    },
+                    {
+                        "rule": "cdn.segment.com/v1/projects/4I2fAtE5IxTlczNT3F91OckHtI5wfR7g/settings",
```
[the remark this answers is not in the document]
Comment (GuiltyDolphin) [reply features/tracker-allowlist.json:429]: @kdzwinel the ID appears to be site-specific - let's only allow the specific request for the time being.

File: features/tracker-allowlist.json (hunk 1 above)
Comment (kdzwinel) [reply features/tracker-allowlist.json:401]: Gotcha, since this is temporary I'm OK with the precise rules.

**Review by kdzwinel — APPROVED** [review]
Not merging per the tag, but LGTM!

**Review by kdzwinel — APPROVED** [review]

**Pull request merged**